IDEAS/Resources/Scripts/mergeAnnex60Script.py

```diff
@@ -10,12 +10,8 @@
         dest_dir = data['dest_dir']
         
 else:
-    print(fileName + " could not be found in your current working directory, please enter source and destination paths. \nThey will be saved for next time. Remove " + fileName + " to reset the paths.")
-    annex60_dir = raw_input("Enter annex60 directory path: \n")
-    dest_dir = raw_input("Enter destination directory path: \n")
-    data = {"annex60_dir":annex60_dir, "dest_dir":dest_dir}
-    with open(fileName, 'w') as dataFile:
-        json.dump(data, dataFile)
+    print(fileName + " could not be found in your current working directory, please enter source and destination paths."
+                     "\nThey will be saved for next time. Remove " + fileName + " to reset the paths.")
```

Comment:
The updated message is not correct, as it requests to enter the source and destination paths. I would change this into "Please check the source and destination path" or something like that.

Reply:
Good catch! I have been too quick for this...

Reply:
https://github.com/open-ideas/IDEAS/pull/1372/commits/6931b6dbe9af0d4ad0cab2fa471ac036891edc64